extension/src/helpers/hooks/useGetBalances.tsx

```diff
@@ -84,8 +84,10 @@ function useGetBalances(options: {
     try {
       const cachedBalanceData =
         cachedBalances[networkDetails.network]?.[publicKey];
+      const isCacheValid =
+        cachedBalanceData && cachedBalanceData.updatedAt > Date.now() - 180000; // 3 minutes
```

Comment:
let's use a similar cache-busting technique for both account-balances and token-prices to make our lives simpler